examples/tests/test_examples.py

```diff
@@ -69,20 +74,34 @@ def test_example_screenshots(module, force_offscreen):
 
     screenshot_path = screenshots_dir / f"{module.stem}.png"
 
+    black = np.zeros(img.shape).astype(np.uint8)
+    black[:, :, -1] = 255
+
+    img = np.array(
+        Image.alpha_composite(
+            Image.fromarray(black),
+            Image.fromarray(img.astype(np.uint8))
+        ).convert("RGB")
+    )
```

Comment:
Not sure if there's a better way to do alpha blending? @almarklein

Reply:
Yes, I this does the trick. Although I think it might be even less code with plain numpy code. This is the `over` operator (https://en.wikipedia.org/wiki/Alpha_compositing), and a lot of its terms disappear because the background is well-defined:
```
a = img[:,:,3]
rgb = img[:,:,:3] * a + background_rgb * (1 - a)
```

With a black bg this simply becomes:
```
rgb = img[:,:,:3] * img[:,:,3]
```

Might be worth double-checking, but I don't think it gets more complicated than this.